actions/admin/routes.go

```diff
@@ -9,7 +9,7 @@ func RegisterRoutes(handlersManager *handlers.Manager) {
 	adminGroupOld := handlersManager.Group(handlers.GroupOldAPI, "/admin")
 
 	adminGroupOld.GET("/stats", handlers.AsAdmin(stats))
-	adminGroupOld.GET("/status", handlers.AsAdmin(status))
+	adminGroupOld.GET("/status", handlers.AsAdmin(statsOld))
```

Comment:
It should be statusOld not statsOld right?
1. Please change statsOld to statusOld
2. or please change the path from /status to /stats

Reply:
thanks for catching this 👍